miden-lib/src/transaction/mod.rs

```diff
@@ -278,5 +295,6 @@ impl TransactionKernel {
             .expect("failed to load miden-lib")
             .with_library(kernel_library)
             .expect("failed to load kernel library (/lib)")
+            .with_debug_mode(true)
```

Comment:
My initial thought was to enable debugging for the tests, because (at least for me) it is common situation when I need to use some debug features during testing. Probably, I can just remove testing mode from here, it's not a big deal.